src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -506,6 +658,100 @@ Sandboxing and Docker dependencies are applicable to the synthetics job manager
   </Collapser>
 </CollapserGroup>
 
+## Sandboxing and Podman dependencies [#sandboxing-and-podman-deps]
+
+Sandboxing and Podman dependencies are applicable to the synthetics job manager in a Podman container system environment.
+
+<CollapserGroup>
+  <Collapser
+    id="podman-dependencies"
+    title={<><img src="/images/synthetic_logo_podman.webp" title="Podman icon" alt="Podman icon" style={{ height: '35px', width: '40px' }}/>Podman dependencies</>}
+  >
+    The synthetics job manager runs in Podman and is able to leverage Podman as a sandboxing technology. This ensures complete isolation of the monitor execution, which improves security, reliability, and repeatability. Every time a scripted or browser monitor is executed, the synthetics job manager creates a brand new Podman container to run it in using the matching runtime.
+
+    The synthetics job manager container needs to be configured to communicate with the Podman engine in order to spawn additional runtime containers. Each spawned container is then dedicated to run a check associated with the [synthetic monitor](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors) running on the private location the synthetics job manager container is associated with.
+
+    There is a crucial dependency at launch. To enable sandboxing, ensure that:
+
+
+
+    1. Install Podman.
+    2. Enable Rootless Execution.
+       * Set up Podman to run containers without requiring root privileges.
+         ```
+         mkdir -p ~/.config/containers
```

Comment:
hope these steps are reviewed by our security team

Reply:
These were the part of google docs shared with the security team initially.